README.md

```diff
@@ -154,6 +154,60 @@ yarn run dev
 ```
 If changes aren't detected, you might need to increase `fs.inotify.max_user_watches`. See <https://webpack.js.org/configuration/watch/#not-enough-watchers>.
 
+#### Deploying a Custom Image to an OpenShift Cluster
+
+Once you have made changes locally, these instructions will allow you to push changes to an
+ OpenShift cluster for others to review.  This involves building a local image, pushing the
+ image to an image registry, then updating the OpenShift cluster to pull the new image.
+
+##### Prerequisites
+
+1. Docker v17.05 or higher
+2. An image registry like [quay.io](https://quay.io/signin/) or [Docker Hub](https://hub.docker.com/)
+
+##### Steps
+1. Create a repository in the image registry of your choice to hold the image
+2. Build Image `docker build -t <your-image-name> [<your-git-url> | <path-to-dockerfile>]`
```

Comment:
It looks like it still says "path-to-dockerfile" when it should be the path to your repository. I'd probably change this to

```suggestion
2. Build image: `docker build -t <your-image-name> .`
```

to keep it simple. Nit: I'd use the same format for the command as you use below

Reply:
idk, for newbies I'd like to understand what the options are.  Unfortunately, I think our documentation is too sparse at times, and assumes knowledge.  I don't see the harm in providing additional info.

Reply:
Personally I think it makes it more complicated for someone new by forcing them to make a choice that is not really important. Most of the time you just want to build the current local branch.

Reply:
Ok, fair enough.  Revised with a little of both of our suggestions.